misc/make-deps.sh

```diff
@@ -75,15 +75,15 @@ fi
 
 if [ $travis -eq 0 ]; then
 	if [ ! -z "$YUM" ]; then
-		if [ -z "$GO" ]; then
+		if [ -z "$GO" ] && [ go version | grep -e 'go1\.[0123456789]\.' -e 'go1\.10\.' ]; then
```

Comment:
Wouldn't `! go version 2> /dev/null` be more what you want??

Reply:
If I'm not mistaken, `[ -z "$GO" ]`does the same thing as ! go version `2>/dev/null` in theory since GO=`command -v go 2>/dev/null`. The issue is that this first condition succeeds when it should not in some installation cases. That's why I added this other condition so we are absolutely sure that go must be installed.

Reply:
I'm just not quite clear what's going on here and why it's needed. Care to discuss in channel or try to explain more please?